file_paths.md

```diff
@@ -275,6 +279,12 @@ create a `puppet` user or group.
         lib *
             puppet.rb *
 
+    C:\Program Files\Puppet Labs\Puppet\pxp-agent *
+        bin *
```

Comment:
We're also currently carrying along a `inc` folder like we did for Facter.  Is this necessary?